.github/workflows/unit.yml

```diff
@@ -0,0 +1,90 @@
+name: unit
+
+on:
+  push:
+    branches: [master]
+  pull_request: # run on all PRs, not just PRs to a particular branch
+
+jobs:
+  # `unit` includes just unit and proto tests.
+  unit:
+    strategy:
+      matrix:
+        node: ['12', '14', '16']
```

Comment:
should we just run on `['12', '16']`? Not sure if there's a lot of downside to running on all three besides more jobs queued up and more chances for flakes